docs/getting_started/installation/gpu/cuda.inc.md

```diff
@@ -38,10 +38,10 @@ We recommend leveraging `uv` to [automatically select the appropriate PyTorch in
 As of now, vLLM's binaries are compiled with CUDA 12.8 and public PyTorch release versions by default. We also provide vLLM binaries compiled with CUDA 12.6, 11.8, and public PyTorch release versions:
 
 ```bash
-# Install vLLM with CUDA 11.8.
-export VLLM_VERSION=0.6.1.post1
-export PYTHON_VERSION=312
-uv pip install https://github.com/vllm-project/vllm/releases/download/v${VLLM_VERSION}/vllm-${VLLM_VERSION}+cu118-cp${PYTHON_VERSION}-cp${PYTHON_VERSION}-manylinux1_x86_64.whl --extra-index-url https://download.pytorch.org/whl/cu118
+# Install vLLM with a specific CUDA version (e.g., 11.8 or 12.6).
+export VLLM_VERSION=0.9.2
+export CUDA_VERSION=118 # or 126
+uv pip install https://github.com/vllm-project/vllm/releases/download/v${VLLM_VERSION}/vllm-${VLLM_VERSION}+cu${CUDA_VERSION}-cp38-abi3-manylinux1_x86_64.whl --extra-index-url https://download.pytorch.org/whl/cu${CUDA_VERSION}
```

Comment:
Why do we sneap Python version 3.8 in here?

Reply:
> Why do we sneap Python version 3.8 in here?

Because the currently released packages all have wheels like "cp38" after the https://github.com/vllm-project/vllm/pull/6394.

As mentioned in https://github.com/vllm-project/vllm/pull/6394#issuecomment-2227336530, the python3 setup.py bdist_wheel --dist-dir=dist --py-limited-api=cp38 command can be installed across various Python versions.

Reply:
Ah I see, good point. This is because `cp38` is the minimum supported version